src/components/auth/RegisterParent.vue

```diff
@@ -169,11 +170,18 @@
         </div>
       </form>
     </div>
+    <div class="flex align-items-center justify-content-center py-3 mt-3 border-top-1 border-gray-200">
```

Comment:
A nitpick, but no need for flex here, a simple `text-center` does the job. More importantly, let's remove that padding (see previous comment)

```suggestion
    <div class="text-center mt-3 border-top-1 border-gray-200">
```